conflowgen/api/truck_arrival_distribution_manager.py

```diff
@@ -18,7 +18,8 @@ def __init__(self):
 
     def get_truck_arrival_distribution(self) -> Dict[int, float]:
         """
-        Returns: The truck arrival distribution.
+        Returns:
+            The truck arrival distribution.
```

Comment:
Please add an empty line between the returns statement and the following text block. The following text block is not what is returned.